python/cuml/test/dask/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from dask_cuda import LocalCUDACluster
+
+from dask.distributed import Client, wait
+
+
+@pytest.fixture(scope="module")
+def cluster():
+
+    print("Starting local cuda cluster for testing")
+    cluster = LocalCUDACluster()
```

Comment:
do we want to leave the n_workers unspecified? it makes me nervous that tests will then work pretty differently when I'm logged into a DGX or a workstation. I'd rather have an explicit "all_gpus_cluster" or something like that for the weird cases when we want to do that.

Reply:
Yeah I was thinking about this when I saw how you parametrized the MNMG random forest tests. Would it make sense for us to have a configuration value that we could set manually and have it use all GPUs otherwise? Maybe we could use the approach we took to our stress & quality test parameters, where we want different environments (like CI vs manually executed tests) to use different sets of parameters?

Reply:
That configurable approach looks like the best way to go forward about this

Reply:
that sounds good to me, like an env var or something that specifies number of gpus for tests? would be nice since then if you have issues with 8 gpus, you can easily rerun with only 1 gpu worker in the same env to test if that's the source of the problem

Reply:
Generally, I've used `CUDA_VISIBLE_DEVICES` to control the number of workers that `LocalCUDACluster` uses. Wouldn't that work here as well (rather than creating a new environment var that essentially does the same thing)?

Reply:
If this is okay, (and assuming CI passes), can we work this into 0.10 as-is? Would you be okay removing the `n_workers` param from random forest tests and using `CUDA_VISIBLE_DEVICES` to control it?

Reply:
I think CUDA_VISIBLE_DEVICES should be ok, but I thought n_workers in rf was so that it could be parameterized to try several different levels?